### ISSUE TITLE
Enhance get-device-rules function to return rules configured via Panorama

### ISSUE THREAD

**Issue by jamesholland-uk** [nautobot/nautobot-plugin-chatops-panorama#111]
### Environment
* Python version:  3.9.9
* Nautobot version: 1.2.8 via Docker Compose
* nautobot-chatops version:  1.6.0
* nautobot-plugin-chatops-panorama version:  1.0.0

### Proposed Functionality
* Currently, the `get-device-rules` feature only returns rules configured locally on the managed firewalls.
* Ideally, as a plugin targeting Panorama specifically, the rules configured in Panorama and pushed to managed devices would also be included in the output from this function.

### Use Case
* Most organisations using Panorama would push rules to firewalls from Panorama, as opposed to configure the rules locally on firewalls. Therefore, the usefulness of the `get-device-rules` would be dramatically increased by enhancing this feature.

**Comment by jdrew82** [comment]
I would agree with this request but upon further investigation of the PAN-OS SDK, it doesn't appear that you can see the rules that are in Panorama. It appears the SDK is focused on working with the firewalls directly using Panorama as a proxy. I think we would need to connect directly to the Panorama instance and run a command at the CLI to get this information.

**Comment by jdrew82** [comment]
I believe this should be included as part of the updates in #120.

**Comment by jdrew82** [comment]
This should be addressed so going to close.
